### PR TITLE
LPS-196918 Generic error attempt publishing a published object

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPS-196918](https://liferay.atlassian.net/browse/LPS-196918)

### CODE REVIEW

**Comment by larissacribeiro** [comment]
ci:test:sf

**Review by alinedoleron — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/EditObjectDetails.tsx [hunk 1]
```diff
@@ -145,13 +145,10 @@ export default function EditObjectDetails({
...
 				if (!publishResponse.ok) {
-					const {title} = (await publishResponse.json()) as {
-						status: string;
-						title: string;
-					};
-
 					openToast({
-						message: title,
+						message: Liferay.Language.get(
+							'object-definition-already-published'
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/EditObjectDetails.tsx:150]: Hi Larissa, we'll need to change this key to adjust the message ok? Thanks :blush: 
```suggestion
							'the-object-definition-has-already-been-published'
```

**Review by alinedoleron — CHANGES_REQUESTED** [review]

**Comment by larissacribeiro** [comment]
ci:test:sf

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:forward:force

**Comment by alinedoleron** [comment]
ci:forward:force

**Comment by alinedoleron** [comment]
ci:forward:force

**Comment by larissacribeiro** [comment]
Resent here in order to try and foward again: https://github.com/liferay-objects/liferay-portal/pull/2704

**Pull request closed without merging**